packages/gravity-ui-web/src/sass/05-components/03-organisms/03-articles/_00-related-items.scss

```diff
@@ -0,0 +1,24 @@
+.grav-c-related-items {
+  max-width: 540px;
```

Comment:
I'd move the `max-width`, `margin-right` and `margin-left` rules down into the `> li { ... }` block.

As per our layout conventions, components shouldn't set explicit sizes on themselves, but they are welcome to do so on their own children.

Also, if we ever enhance this in the future to show successive links side-by-side, it'll be easier since we just need to layout the `li`s and don't need to fiddle with the enclosing `ul`'s size.